sg20/d1389/d1389.bs

```diff
@@ -323,27 +323,46 @@ TODO (why?)
 
 TODO (why?)
 
-### [tools.debugger] Introduce a debugger early
+### [tools.testing] Introduce a testing framework
+
+Examples: [Catch2](https://github.com/catchorg/Catch2), [Google Test](https://github.com/google/googletest)
 
 TODO (why?)
 
+### [tools.debugger] Introduce a debugger early
+
+The ability to step through running code and examine execution state will enable students to troubleshoot issues and correct assumptions they have made about the behavior of language and library features.
```

Comment:
Should there be recommendations for debuggers? I'm only aware of four:
* GDB
* LLDB
* Visual Studio's debugger
* windbg (not for beginners?)

Are there others that should be considered or avoided?